Cargo.toml

```diff
@@ -253,6 +255,10 @@ required-features = ["draw"]
 name = "log_basic"
 required-features = ["log"]
 
+[[example]]
+name = "render_instancing_cubes"
```

Comment:
This example's name is `renderer_instancing_cubes`

Reply:
Done!